README.md

```diff
@@ -51,6 +51,10 @@ Copy `config/config-example.js` into `config/config.js`, and edit as you please.
 
 Congratulations, you're done setting up Pokémon Showdown.
 
+After you will need to install the dependencies:
+
+    npm install
```

Comment:
Anyway, this should ideally be fixed by fixing the code to auto-install dependencies, not by amending the readme.